README.md

```diff
@@ -114,6 +115,40 @@ connection.execute(
   }
 );
 ```
+
+## Using connection pools
+
+Connection pools provide automatic ways of managing multiple connections. A pool maintains a queue of *free* connections, allowing the use of multiple connections in parallel.
```

Comment:
The wording is not correct here.

A better way of describing them would be:
"Connection pools help reduce the time spent connecting to the MySQL server by reusing a previous connection, leaving them open instead of closing when you are done with them.

This improves the latency of queries as you avoid all of the overhead that comes with establishing a new connection."